portal/src/main/webapp/js/src/patient-view/util/util.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/**
+ * Get first key found in object. Otherwise return null.
+ * @param {object} object - object with key/value pairs
+ * @param {array} keys - array of keys
+ */
+window.getFirstKeyFound = function(object, keys) {
```

Comment:
Thanks, yeah I think `getFirstKeyFound` might be general enough to move to `cbio-util.js`